lib/ClangImporter/ClangImporter.cpp

```diff
@@ -716,6 +716,8 @@ addCommonInvocationArguments(std::vector<std::string> &invocationArgStrs,
     invocationArgStrs.push_back(importerOpts.IndexStorePath);
   }
 
+  invocationArgStrs.push_back("-fansi-escape-codes");
```

Comment:
This should be conditionalized on DiagnosticOptions::UseColor, no? Or will it just not use the escape codes if they never come up?

Reply:
Exactly, if color is emitted, it'll use ansi escapes instead of SetConsoleMode, but it doesn't actually trigger the use of color.